codecov.yml

```diff
@@ -0,0 +1,20 @@
+codecov:
+  branch: dev
+
+coverage:
+  status:
+    project:
+      default:
+        # minimum coverage ratio that the commit must meet to be considered a success
+        target: 40%
+        if_ci_failed: error
+        only_pulls: true
+    patch:
+      default:
+        # minimum coverage ratio that the commit must meet to be considered a success
+        target: 40%
+        if_ci_failed: error
+        only_pulls: true
+
```

Comment:
Should we use this: https://docs.codecov.com/docs/common-recipe-list#only-comment-on-coverage-changes ?

Reply:
I've took another look at this, and it doesn't seem so necessary as I thought, since Codecov doesn't create new comments every new commit.

Reply:
Good point. It would be useful, but only in edge cases where the coverage becomes zero for some reason.